contracts/MintingHubV2/Position.sol

```diff
@@ -199,9 +202,10 @@ contract Position is Ownable, IPosition, MathUtil {
      */
     function initialize(address parent, uint40 _expiration) external onlyHub {
         if (expiration != 0) revert AlreadyInitialized();
-        if (_expiration < block.timestamp || _expiration > Position(original).expiration()) revert InvalidExpiration(); // expiration must not be later than original
+        if (_expiration < block.timestamp || _expiration > Position(parent).expiration()) revert InvalidExpiration();
```

Comment:
@lapatric FYI

Explanation:

When a position is cloned, it should inherit constraints from its parent position, not necessarily the original position from which the entire structure might have originated. By changing:

Position(original)

to:

Position(parent)

we ensure that the new clone respects the same expiration boundaries as its immediate parent. The parent variable represents the position that is currently being cloned. If the clone tried to set an expiration date beyond what the parent position allows, it would violate the intended constraint system. Therefore, referencing parent instead of original is the correct and logical choice in this scenario.

Reply:
I'm not sure if this change really belongs in this PR, because I don't think it has anything to do with the actual topic here. 

I would be happy to receive feedback.

Reply:
Keep unchanged. Since V2 operates with global states for all their "family childs" 

E.g.
* works with minting limit 
* works with expiration

```
if (_expiration < block.timestamp || _expiration > Position(original).expiration()) revert InvalidExpiration(); // expiration must not be later than original
```